src/hooks/UseQuayConfig.ts

```diff
@@ -10,9 +10,17 @@ export function useQuayConfig() {
 
   useEffect(() => {
     (async () => {
-      const config = await fetchQuayConfig();
-      setQuayConfig(config);
-      console.log(config);
+      // NOTE: Making the decision that loading the Quay configuration
+      // is not required. If the load fails the app will continue running.
+      // Components using this hook should check for null values. This
```

Comment:
I wonder if we should call this out in the readme? Config is already an odd call, maybe we should call it out in the future explaining why we even make the call.

Reply:
Yeah good point. I'm not sure if the readme is the right place for this info. Would that go into a design details section or something along those lines? Maybe something to discuss separately